analysis/combiners.py

```diff
@@ -263,12 +281,32 @@ class CountCombiner(SumCombiner):
     # expected_cross_partition_error, var_cross_partition_error)
     AccumulatorType = Tuple[float, float, float, float, float]
 
-    def create_accumulator(self, data: PreaggregatedData) -> AccumulatorType:
-        count, _sum, n_partitions = data
-        data = count, count, n_partitions
-        self._params.aggregate_params.min_sum_per_partition = 0.0
-        self._params.aggregate_params.max_sum_per_partition = self._params.aggregate_params.max_contributions_per_partition
-        return super().create_accumulator(data)
+    def create_accumulator(
```

Comment:
It seems we are duplicating logic again. Why don't we rely on super().create_accumulator()?

Reply:
Thanks a lot! This change in this PR was made before your refactoring )), of course we should use super().create_accumulator(), done